afs/templates/views/components/workflows/create-project-workflow/add-things-step.htm

```diff
@@ -57,7 +60,7 @@
                         <div class="wf-multi-tile-card-info-details">
                         <dt data-bind="text: $data.displayname"></dt>
                         <dd>
-                            <a data-bind="text: $data.displaydescription"></a>
+                            <a data-bind="text: $($data.displaydescription).text() || 'No description'"></a>
```

Comment:
This doesn't seem to be working, almost always return "No description"

Reply:
Are you sure that your objects have a description statement? It seems to be working okay for me:

![image](https://user-images.githubusercontent.com/1877663/135695765-a03473be-fcaf-4225-9c9e-0e47bc5909da.png)

Reply:
<img width="1183" alt="Screen Shot 2021-10-01 at 4 37 58 PM" src="https://user-images.githubusercontent.com/17627335/135696213-007f6e2e-8778-4766-8808-eb8f6f78d4bd.png">

Reply:
$() may not be working on a simple string.

Reply:
I replaced the jQuery text method for the ko html binding. It should work for you now.

Reply:
yep, html binding works.